### PR TITLE
Fix log group memory leak when using Golang plugins and reduce alarm frequency in LOGSTORE send mode

### PR DESCRIPTION
**Pull request by linrunqi08**
1. Fixed the issue where the logGroup pointer was not released when using Golang plugins. 
2. Increase alarm threshold and reduce 'package list interval' configuration to fix the alarm of LOG_GROUP_WAIT_TOO_LONG_ALARM in LOGSTORE mode.

### CODE REVIEW

File: core/processor/daemon/LogProcess.cpp [hunk 1]
```diff
@@ -434,6 +433,10 @@ void* LogProcess::ProcessLoop(int32_t threadNo) {
                         "parse_failures", profile.parseFailures)("parse_time_failures", profile.parseTimeFailures)(
                         "regex_match_failures", profile.regexMatchFailures)("history_failures", profile.historyFailures));
             }
+            for (auto pLogGroup : logGroupList) {
```
Comment (messixukejia) [inline core/processor/daemon/LogProcess.cpp:436]: 智能指针

File: core/processor/daemon/LogProcess.cpp [hunk 2]
```diff
@@ -434,6 +433,10 @@ void* LogProcess::ProcessLoop(int32_t threadNo) {
...
+                delete pLogGroup;
+            }
+            logGroupList.clear();
```
Comment (messixukejia) [inline core/processor/daemon/LogProcess.cpp:439]: 历史修复

**Review by messixukejia — CHANGES_REQUESTED** [review]

File: core/processor/daemon/LogProcess.cpp (hunk 1 above)
Comment (linrunqi08) [reply core/processor/daemon/LogProcess.cpp]: done

File: core/processor/daemon/LogProcess.cpp (hunk 2 above)
Comment (linrunqi08) [reply core/processor/daemon/LogProcess.cpp:435]: ASAN扫描出来的都是new Thread的问题，这个单独一个pr去修吧

**Review by yyuuttaaoo — APPROVED** [review]

**Review by messixukejia — APPROVED** [review]

**Pull request merged**